docs/user_guide/fl_simulator.rst

```diff
@@ -29,14 +29,16 @@ Command Usage
     optional arguments:
     -h, --help            show this help message and exit
     -w WORKSPACE, --workspace WORKSPACE
-                            WORKSPACE folder
+                        WORKSPACE folder
```

Comment:
These changes to the spacing will cause the final built page to have hard to read spacing.

Reply:
If it is okay with you, I can revise this and then add it as a commit directly to this file.

Reply:
Please make the changes necessary as we discussed.